python/GafferImageUI/ConstantUI.py

```diff
@@ -66,6 +66,16 @@
 
 		],
 
+		"layer" : [
+
+			"description",
+			"""
+			The layer to generate. The output channels will
+			be named ( layer.R, layer.G, layer.B and layer.A ).
```

Comment:
I'm assuming that while this is true at the data level, for now the user has no way of visually interacting with any channels that have a non-empty layer, right? Should we maybe hide the plug from the UI until they do?